node/hack/governor/check_query.go

```diff
@@ -0,0 +1,20 @@
+// This tool can be used to confirm that the CoinkGecko price query still works after the token list is updated.
+// Usage: go run check_query.go
+
+package main
+
+import (
+	"github.com/certusone/wormhole/node/pkg/governor"
+
+	"go.uber.org/zap"
+)
+
+func main() {
+	logger, _ := zap.NewDevelopment()
+	logger.Info("Testing Coin Gecko query.")
+	if err := governor.CheckQuery(logger); err != nil {
+		logger.Fatal("Coin Gecko query failed", zap.Error(err))
+	}
```

Comment:
Would this make more sense to make this part of the unit-tests instead? or is the intent that this would be run by a Guardian operator?

Reply:
No this is in the right place. It’s hitting a real live service not a mock so it isn’t a unit test. This is what the hack package is for. Simple scripts to ensure things are good.